pkg/neotest/signer.go

```diff
@@ -44,6 +44,14 @@ type MultiSigner interface {
 	Single(n int) SingleSigner
 }
 
+// ContractSigner is an interface for contract signer.
+// `InvocationScript()` and `ScriptHash()` must be defined.
+type ContractSigner interface {
+	Signer
+	// InvocationScript returns an invocation script to sign contract.
+	InvocationScript() []byte
```

Comment:
I'm not sure it's a good idea to have constant contract invocation script in the `ContractSigner` implementation, because it's not a private-key-based account, the script itself can be different from invocation to invocation, maybe we need some function for it to be provided via `NewContractSigner` constructor. But probably for tests it's sufficient, at least it allows to test contract signers. @roman-khimov, what do you think?

Reply:
In our tests signer encompasses all the context (`Verify` accepts some address of `interop.Hash160` and checks whether it is in the "trusted list").
For different address we can create a different instance of a signer. `InvocationScript()` method seems the most flexible option, but it doesn't cancel the utility of `NewContractSigner` helper.

Reply:
Fixed.